app/models/block.rb

```diff
@@ -0,0 +1,6 @@
+class Block < ActiveRecord::Base
+  has_many :cards, dependent: :destroy
+  belongs_to :user
+
+  validates :title, presence: { message: 'Необходимо заполнить поле.' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.